cryptography/src/bls12381/primitives/group.rs

```diff
@@ -322,8 +324,20 @@ impl Hash for Scalar {
     }
 }
 
+impl Debug for Scalar {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", hex(&self.as_slice()))
+    }
+}
+
+impl Display for Scalar {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", hex(&self.as_slice()))
+    }
+}
```

Comment:
To prevent writing twice, should we have one be the primary (say `Debug`) and have `Display` call `Debug::fmt(self, f)`?

Reply:
This is a pattern I adapted from elsewhere in `cryptography`. Happy to migrate to a shared implementation but think we should do all at once.